### PR TITLE
View competitions

### PR DESCRIPTION
**Pull request by mthembbu**

### CODE REVIEW

File: skietbaan_fe/src/components/ViewComp.js [hunk 1]
```diff
@@ -1,5 +1,7 @@
 import React, { Component } from 'react';
-import { fetchcomp, updateByIDcomp } from '../actions/competition.action';
+import { fetchcomp,
```
Comment (FredJEhlers) [inline skietbaan_fe/src/components/ViewComp.js]: Fix the indentation
```
import {
 fetchcomp,
 updateByIDcomp 
} from '../actions/competition.action';
```

File: skietbaan_fe/src/scss/create-page.css [hunk 2]
```diff
@@ -1,11 +1,23 @@
+.view-header{
+    position: center;
+	text-align: center;
```
Comment (FredJEhlers) [inline skietbaan_fe/src/scss/create-page.css]: indentation

File: skietbaan_fe/src/components/ViewComp.js (hunk 1 above)
Comment (mthembbu) [reply skietbaan_fe/src/components/ViewComp.js]: Thought, I double checked. Will resolve

File: skietbaan_fe/src/actions/competition.action.js [hunk 3]
```diff
@@ -1,8 +1,8 @@
 import { NEW_COMP,
```
Comment (FredJEhlers) [inline skietbaan_fe/src/actions/competition.action.js]: NEW_COMP on the new line as well I showed you in the comment

File: skietbaan_fe/src/actions/types.js [hunk 4]
```diff
@@ -54,4 +54,4 @@ export const PASS_ID = 'PASS_ID';
 export const GETNAME = 'GETNAME';
```
Comment (FredJEhlers) [inline skietbaan_fe/src/actions/types.js]: Constants words to be separated by an underscore `GET_NAME `

Comment (FredJEhlers) [reply skietbaan_fe/src/actions/types.js]: empty line EOF

File: skietbaan_fe/src/components/ViewComp.js (hunk 1 above)
Comment (FredJEhlers) [reply skietbaan_fe/src/components/ViewComp.js]: look at my import again please you have not done as requested

File: skietbaan_fe/src/components/ViewComp.js [hunk 5]
```diff
@@ -34,38 +34,37 @@ class ViewComp extends Component {
 	changeStatus(status, i) {
 		const newCompOBJArr = [ ...this.props.compOBJ ]; //cloning an array of competition
 		newCompOBJArr[i].status = !status; //changing the status of a newly cloned array
-		console.log('data status',newCompOBJArr[i].status)
-		this.props.updateByIDcomp(newCompOBJArr[i], i); //updating the competition status by ID in the back-end
+		console.log('data status', newCompOBJArr[i].status);
```
Comment (FredJEhlers) [inline skietbaan_fe/src/components/ViewComp.js]: remove debug log

File: skietbaan_fe/src/components/ViewComp.js [hunk 6]
```diff
@@ -34,38 +34,37 @@ class ViewComp extends Component {
...
+		this.props.updateByIDcomp(newCompOBJArr[i], i + 1); //updating the competition status by ID in the back-end
```
Comment (FredJEhlers) [inline skietbaan_fe/src/components/ViewComp.js]: incorrect case should be `updateByIdComp`

File: skietbaan_fe/src/scss/createcomp.css [hunk 7]
```diff
@@ -1,98 +1,114 @@
...
-
-input.invalid{
-  border:1px solid red;
+input.invalid {
+	border: 1px solid red;
 }
-.invalid-message{
-  color:red;
-  font-size: 12px;
+.invalid-message {
```
Comment (FredJEhlers) [inline skietbaan_fe/src/scss/createcomp.css]: space between css classes

Comment (FredJEhlers) [reply skietbaan_fe/src/scss/createcomp.css]: cover all instances

File: skietbaan_fe/src/components/ViewComp.js (hunk 5 above)
Comment (mthembbu) [reply skietbaan_fe/src/components/ViewComp.js]: Eish, yeah. Forgot that one..

**Pull request merged**